tools/ci_build/github/linux/docker/scripts/install_onnx.sh

```diff
@@ -16,21 +19,15 @@ for onnx_version in ${!version2tag[@]}; do
     echo "first pass";
   else
     echo "deleting old onnx-${lastest_onnx_version}";
```

Comment:
This  file was added in PR #1073, but indeed it doesn't really work. It has multiple problems.  Even after this checkin, it is still broken.

Reply:
It is passing CI -- is it broken in some other way? It seems to be cycling through different version of ONNX test data successfully in runs.

Reply:
But, the data structure of the "version2tag" is an unordered map. So this piece of code is non-deterministic. You can't predict which version of onnx will be kept after running

Reply:
I'll talk to Randy, and we'll address it later.